### PR TITLE
auto charge

### PR DESCRIPTION
**Pull request by zeitlinger**
auto-decline charge if player can charge fully during income phase && player has passed

### CODE REVIEW

**Comment by zeitlinger** [comment]
I want to add some more auto-charge features, but I'm not sure how far I'll get.

File: src/player.spec.ts [hunk 1]
```diff
@@ -75,4 +76,89 @@ describe("Player", () => {
...
+      {
+        name: "events - will not charge fully (brainstone)",
+        power: new Power(),
+        brainstone: BrainstoneArea.Area2,
+        events: Event.parse(["+2pw", "+1t"], Command.ChooseIncome),
+        expected: false,
+      },
+    ];
+
+    tests.forEach((test) => {
```
Comment (coyotte508) [inline src/player.spec.ts:153]: I prefer `for (const test of tests) {` unless there's a compelling reason not to do it that way. It's more natural / works with async.

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "build": "npm run clean && tsc",
     "clean": "rm -Rf dist",
     "lint": "eslint . --ext .ts",
+    "fix-lint": "eslint . --ext .ts --fix",
```
Comment (coyotte508) [inline package.json:25]: Not needed - you can just do `yarn lint --fix`.

**Review by coyotte508 — CHANGES_REQUESTED** [review]
Thank you for the clean up!

**Comment by zeitlinger** [comment]
> Thank you for the clean up!

linter was complaining..

**Review by coyotte508 — APPROVED** [review]

**Pull request merged**